xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/xwiki.js

```diff
@@ -253,8 +253,8 @@ Object.extend(XWiki, {
                       });
 
                       // Replace the element's content with the temporary container's content, while also evaluating any inline scripts.
-                      // Note: This also does stripScripts internally, but it's OK now, since we have already added the necessary external scripts to head.
-                      $(extraID + "pane").update(container.innerHTML);
+                      // Note: This also allows script tag defined in the update html to be loaded.
+                      $(extraID + "pane").update(container.descendants()[0]);
```

Comment:
Typos: "Note: This also allows script tags defined in the updated html to be loaded."

Reply:
Thanks, typo fixed here https://github.com/xwiki/xwiki-platform/pull/1315/files#diff-3c136b7fe6897859a417c7b58211182dR256